.github/workflows/push.yml

```diff
@@ -277,6 +305,58 @@ jobs:
           UNAFFECTED=$(./scripts/ci/list-unaffected.sh docker-next docker-express docker-static docker-cypress)
           echo "::set-output name=UNAFFECTED::$UNAFFECTED"
 
+  tests:
```

Comment:
Consider moving this to a [composite action](https://docs.github.com/en/actions/creating-actions/creating-a-composite-action)

Reply:
@sindrig I find it disappointing that composite actions do not support conditional steps. I guess you can use scripting when having a shell command but kind of hard when you are `use`-ing an action.